src/mon/OSDMonitor.cc

```diff
@@ -4387,22 +4389,33 @@ bool OSDMonitor::preprocess_command(MonOpRequestRef op)
 	  case CSUM_TYPE:
 	  case CSUM_MAX_BLOCK:
 	  case CSUM_MIN_BLOCK:
-            pool_opts_t::key_t key = pool_opts_t::get_opt_desc(i->first).key;
-            if (p->opts.is_set(key)) {
-              f->open_object_section("pool");
-              f->dump_string("pool", poolstr);
-              f->dump_int("pool_id", pool);
-              if(*it == CSUM_TYPE) {
-                int val;
-                p->opts.get(pool_opts_t::CSUM_TYPE, &val);
-                f->dump_string(i->first.c_str(), Checksummer::get_csum_type_string(val));
-              } else {
-                p->opts.dump(i->first, f.get());
-              }
-              f->close_section();
-              f->flush(rdata);
-            }
+	    {
+	      pool_opts_t::key_t key = pool_opts_t::get_opt_desc(i->first).key;
+	      if (p->opts.is_set(key)) {
+		f->open_object_section("pool");
+		f->dump_string("pool", poolstr);
+		f->dump_int("pool_id", pool);
+		if(*it == CSUM_TYPE) {
+		  int val;
+		  p->opts.get(pool_opts_t::CSUM_TYPE, &val);
+		  f->dump_string(i->first.c_str(), Checksummer::get_csum_type_string(val));
+		} else {
+		  p->opts.dump(i->first, f.get());
+		}
+		f->close_section();
+		f->flush(rdata);
+	      }
+	    }
```

Comment:
There is a pool_opts_t :: key_t key declaration in the switch statement, which requires parentheses for compilation.
The pool_opts_t :: key_t key declaration has been removed from the switch statement to avoid parentheses.